.travis/memcached.ini

```diff
@@ -0,0 +1 @@
+extension=memcached.so
```

Comment:
This file shouldn't be necessary since we're using github actions and not TravisCI for builds